app/src/main/AndroidManifest.xml

```diff
@@ -32,6 +32,8 @@
         android:allowBackup="false"
         android:icon="@drawable/ic_launcher"
         android:label="@string/app_name"
+        android:hardwareAccelerated="false"
+        android:largeHeap="true"
```

Comment:
@Sam1301 There was an issue outOfMemory error with loading the images. So i added largeheap  to optimize using memory. :)